glass/grf/_solver.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+from transformcl import cltocorr, corrtocl
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from numpy.typing import NDArray
+
+from ._core import Transformation, corr, dcorr, icorr
```

Comment:
Absolute imports should be preferred if relative imports are not a necessity
```suggestion
from glass.grf._core import Transformation, corr, dcorr, icorr
```